README.md

```diff
@@ -450,6 +452,48 @@ else {
 }
 ```
 
+## `OIDC` token.parseAndStoreTokensFromUrl(tokenStorageKeys)
+
+Parse and **store** the access or ID Tokens from the url after a successful authentication redirect. This is used when `authParams.display = 'page'`.
+
+* `tokenStorageKeys` - Override the default storage naming conventions for the access and/or ID tokens. Defaults to `accessToken` and `idToken`.
+
+```javascript
+var signIn = new OktaSignIn({
+  baseUrl: 'https://{yourOktaDomain}',
+  clientId: '{{myClientId}}',
+  redirectUri: '{{redirectUri configured in OIDC app}}',
+  authParams: {
+    responseType: 'id_token',
+    // `display: page` will initiate the OAuth2 page redirect flow
+    display: 'page'
+  }
+});
+
+// The user has just landed on our login form, and has not yet authenticated
+// with a Social Auth IDP.
+if (!signIn.token.hasTokensInUrl()) {
+  signIn.renderEl({el: '#osw-container'});
+}
+
+// The user has redirected back after authenticating and has their access or
+// ID Token in the URL.
+else {
+  signIn.token.parseAndStoreTokensFromUrl()
+  .then(function() {
+    const idToken = signIn.tokenManager.get('idToken');
+    console.log(idToken);
+  });
+
+  // Optionally - use a different key for the idToken
+  signIn.token.parseAndStoreTokensFromUrl({ idToken: 'id-token-key' })
```

Comment:
Should we be updating our example code in https://github.com/okta/okta-signin-widget#rendereloptions-success-error to use this new method?

Reply:
Ideally, yes. However I'm going to hold off on doing that for now - at least until these changes get published externally.

Due to the way we publish the Widget, we need to be careful about including README changes for functionality that isn't available on both the CDN and registry.